### PR TITLE
feat(emoji): add docs page for emoji component

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
Docs page for Emojis  as of https://github.com/fomantic/Fomantic-UI/pull/326

## Screenshots
emoji1
emoji1a
image
emoji2
emoji3
emoji4
emoji5

### CODE REVIEW

File: server/files/stylesheets/docs.css [hunk 1]
```diff
@@ -1541,6 +1541,23 @@ body.progress.animated .ui.progress .bar {
   position: relative;
 }
 
+/*--------------
+      Emoji
+---------------*/
+#example emojiSearch.ui.search {
```
Comment (exoego) [inline server/files/stylesheets/docs.css]: Missing dot?
```suggestion
#example .emojiSearch.ui.search {
```

Comment (lubber-de) [reply server/files/stylesheets/docs.css]: No, it is referring to the implemented tag. It's not a class (i copied the logic from the icon module)
Related HTML code is in emoji.html.eco
```
    <emojiSearch class="ui right aligned search">
      <div class="ui icon input">
        <input class="prompt" type="text" placeholder="Search emojis...">
        <i class="search icon"></i>
      </div>
      <div class="results"></div>
    </emojiSearch>
```

Comment (exoego) [reply server/files/stylesheets/docs.css]: Ah, I see.

File: server/documents/elements/emoji.html.eco [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+    <h2 class="ui header">Emoji Set</h2>
+    <p>An emoji set contains all <a href="https://unicode.org/emoji/charts/full-emoji-list.html">Unicode 12.1 Emoji Characters</a></p>
+    <div class="ui ignored warning message">
+      Emojis serve a very similar function to text in a page. In Fomantic emojis receive a special tag <code>&lt;em&gt;</code> which allow for an abbreviated markup when sitting along-side text.
+        <br>
+        Because of the complexity and possible interferance with existing fomantic classnames, emoji names have to be provided via the <code>data-emoji</code> attribute instead of usual classnaming.
+    </div>
+    <div class="ui ignored message">
+        By default all emoji images are linked to CDN and are <b>not part of FUI itself</b>.
```
Comment (hammy2899) [inline server/documents/elements/emoji.html.eco]: Should read "By default all emoji images are linked to a CDN and are not part of Fomantic itself"

File: server/documents/elements/emoji.html.eco [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          <ul class="ui list">
+              <li>The default theme links to the set by <a href="https://twemoji.twitter.com/">Tweemoji v12</a> under CC-BY 4.0 license</li>
+              <li>As an alternative the available joypixels theme links to the Emoji set designed by <a href="https://www.joypixels.com/emoji/">Joypixels 5</a> under their <a href="https://www.joypixels.com/licenses/free">free license model</a></li>
+          </ul>
+          If needed, you might download your personal icon set from the respective websites respecting their individual licenses, host it locally and change the <code>@emojipath</code> in <code>emoji.variables</code> of the theme you are using.
```
Comment (hammy2899) [inline server/documents/elements/emoji.html.eco]: `@emojipath` should be `@emojiPath` as per the code https://github.com/fomantic/Fomantic-UI/blob/163d3e2a070ecfe2672dc009c23b93e246b3c279/src/themes/twitter/elements/emoji.variables#L8

Comment (lubber-de) [reply server/documents/elements/emoji.html.eco]: Done in fafc1edc76e8f5256dc395224881ee7bccd48efd 🙂

File: server/documents/elements/emoji.html.eco (hunk 2 above)
Comment (lubber-de) [reply server/documents/elements/emoji.html.eco]: Done in fafc1edc76e8f5256dc395224881ee7bccd48efd 🙂

**Pull request merged**